handler.go

```diff
@@ -66,6 +66,13 @@ var (
 	ioCloserType           = reflect.TypeOf((*io.Closer)(nil)).Elem()
 )
 
+// AddHandlers adds all the handlers in the given slice to mux.
+func AddHandlers(mux *httprouter.Router, hs []Handler) {
```

Comment:
I think I'd prefer the router to be called "r", just because an http.ServeMux is a different thing.

Reply:
done